torch/_functorch/vmap.py

```diff
@@ -519,6 +533,8 @@ def wrap_batched(args, bdims, level):
 
 
 def unwrap_batched(args, level):
+    if has_torch_function(args):
+        return handle_torch_function(unwrap_batched, args, args, level)
```

Comment:
Why do we need this one? I would have hoped that the vmap_impl extension point would be sufficient

Reply:
We don't, we can remove that.